libs/vkd3d/meson.build

```diff
@@ -45,8 +45,8 @@ if not enable_standalone_d3d12
     dependencies        : [ vkd3d_common_dep, vkd3d_shader_dep ] + vkd3d_extra_libs,
     include_directories : vkd3d_private_includes,
     install             : true,
-    vs_module_defs      : 'vkd3d.def',
     version             : '1.1.0',
+    c_args              : '-DVKD3D_EXPORTS',
```

Comment:
Ok, it's defined here ... Any reason why we would be building without VKD3D_EXPORTS, does it need to be a global define at all?

Reply:
We need to define it when building as a DLL as opposed to importing it because of the differences between ``dllexport`` and ``dllimport`` declspec on Windows.

When building as an SO, this doesn't do anything -- it will just always mark the symbols as visible.